src/subscript/field_statistics/field_statistics.py

```diff
@@ -163,26 +203,41 @@
 
 -- Installation of the ERT workflow:
 DEFINE <FIELD_STAT_CONFIG_FILE>  ../input/config/field_param_stat.yml
--- The workflow job FIELD_STATISTICS is generated automatically by ERT
--- The workflow file wf_field_param_statistics run FIELD_STATISTICS
+-- The workflow job FIELD_STATISTICS is generated automatically by ERT.
+-- The workflow file wf_field_param_statistics run FIELD_STATISTICS.
 LOAD_WORKFLOW           ../../bin/workflows/wf_field_param_statistics
 
--- The workflow file to be located under ert/bin/workflows run FIELD_STATISTICS:
--- Example of a workflow file can be
+-- Example of a workflow file to be located under ert/bin/workflows which 
+-- run FIELD_STATISTICS:
 FIELD_STATISTICS -c <FIELD_STAT_CONFIG_FILE>
                  -p <CONFIG_PATH>
                  -e <SCRATCH>/<USER>/<CASE_DIR>
                  -r <RESULT_PATH>
--- where <FIELD_STAT_CONFIG_FILE> is the usre specification for this script,
--- and where <CONFIG_PATH> is the ERT <CONFIG_PATH> for the ERT project,
--- and where ensemble directory is specified by  the '-e' option and
--- where the result directory is specified by the <RESULT_PATH>. This is optional
--- since share/grid_statistics is used as default.
--- The results from iter-0 is also copied to 'realization-0/iter-0/share/results/grids
--- and results from iter-3 is copied to 'realization-0/iter-3/share/results/grids.
--- Workflow job for ERT to calculate field statistics is automatically
--- generated by ERT from the subscript repository, but when setting it 
--- up manually, it look like this:
+                 -z <Name of output generated python script for RMS>
+                 -g
+
+where <FIELD_STAT_CONFIG_FILE> is the user specification for this script,
+and where <CONFIG_PATH> is the ERT <CONFIG_PATH> for the ERT project,
+and where ensemble directory is specified by  the '-e' option and
+where the result directory is specified by the <RESULT_PATH>. This is optional
+since share/grid_statistics is used as default.
+The output python script is meant to be used in a RMS job to
+load the results into RMS. The default is to write the generated script
+to the file 'tmp_import_ensemble_field_statistics.py' in the result directory.
+The option '-g' will copy the result into initial and final realization
+of the geogrid under realization-0 in the directories:
+
+    realization-0/iter-0/share/results/grids/geogrid--<resultparameter>.roff
+    realization-0/iter-3/share/results/grids/geogrid--<resultparameter>.roff
+
+where iter-0 is initial ensemble and iter-3 is final ensemble.
+The default (when this option is not specified) is not to copy results into
+one realization of the geogrid.
+
+Workflow job for ERT to calculate field statistics FIELD_STATISTICS 
```

Comment:
As discussed off-line, I think it is important to state in the documentation that the averaging has weknesses when vertical number of grid cells are varying for realization to realization. State somehow that it is "approximative".

Reply:
Agree.